internal/client/client.go

```diff
@@ -48,6 +48,15 @@ type TemporaryCredentials struct {
 	SessionToken    string `json:"session_token"`
 }
 
+type AccountDetails struct {
+	AccountName   string `json:"account_name"`
+	AccountNumber string `json:"account_number"`
+	AccountTypeID int64  `json:"account_type_id"`
+	CreatedAt     string `json:"created_at"`
```

Comment:
Yep, agreed. I choose to keep those fields since they might be useful in the future for other use-cases, but definitely not needed here and better to omit them to make this cleaner.